clippy_lints/src/transmute/transmute_slice_to_larger_element_type.rs

```diff
@@ -0,0 +1,71 @@
+use super::TRANSMUTE_SLICE_TO_LARGER_ELEMENT_TYPE;
+use clippy_utils::diagnostics::span_lint_and_then;
+use clippy_utils::source::reindent_multiline;
+use clippy_utils::sugg;
+use clippy_utils::ty::approx_ty_size;
+use rustc_errors::Applicability;
+use rustc_hir::Expr;
+use rustc_lint::LateContext;
+use rustc_middle::ty::{self, Ty};
+use std::borrow::Cow;
+
+// TODO: Adjust the parameters as necessary
```

Comment:
What is there left to adjust?

Reply:
There's nothing, I've forgot to remove :c

Reply:
fixed in [3bf3cdf](https://github.com/rust-lang/rust-clippy/pull/10312/commits/3bf3cdfdf3f5041592d9403b7acf583fad1a7018)